packages/editor/src/components/editor/canvas/Draggable.css

```diff
@@ -1,14 +1,17 @@
 .draggable {
   padding: var(--draw-padding, var(--size-2) var(--size-4));
-  border: var(--select-border-size, 1px) solid transparent;
+  border: var(--select-border);
+  border-radius: var(--border-r2);
```

Comment:
not sure if every block should have this border. in the mock it looks to me, that only the layout has a border (otherwise also the UserId and Id blocks should have a border).

Reply:
Yes, you're right, but I think it's better for the user if you have a frame for all components (at least in edit mode). That way you can see much more quickly what you can move or not?
In addition, it is currently somewhat more complex to implement, because as far as I can tell there is currently no way to distinguish between layout components and "normal components" in the draggable.

Reply:
also ok for me :)
just saw it in the mock, ;)

Reply:
I will certainly make a note of this point and possibly discuss it again with anais. But for now, I would leave it as it is :) 👍